src/chain_sync/tipset_syncer.rs

```diff
@@ -862,16 +862,14 @@ async fn sync_headers_in_reverse<DB: Blockstore + Clone + Sync + Send + 'static,
     tracker.write().set_epoch(current_head.epoch());
 
     let total_size = proposed_head.epoch() - current_head.epoch();
-    let pb = ProgressBar::new(total_size as u64);
-    pb.message("Downloading headers ");
-    pb.set_max_refresh_rate(Some(std::time::Duration::from_millis(500)));
+    let mut wi = wrap_iter("Downloading headers", 0..total_size);
```

Comment:
Can't use an iterator here, unfortunately. Epochs are not guaranteed to be consecutive.

Reply:
Ah yes, ok will go back to previous design.